src/scss/components/_sidenav.scss

```
@@ -34,3 +34,14 @@
   padding-top: units(1.5);
   padding-bottom: units(1.5);
 }
+
+li > .active {
+  color: $theme-color-primary-dark;
+  font-weight: bold;
+}
```

Since the Login design system extends the USWDS side navigation component, what do you think about applying the styling to those existing classes, rather than creating new ones?

We already add some custom styles above for `usa-current`, so we might have to "undo" those to get the effect we want with the `.active` class here:

```suggestion
.usa-sidenav__sublist .usa-current {
  @include remove-bar;
  background-color: color('white');
  color: $theme-color-primary-dark;
  font-weight: bold;
}
```